### PR TITLE
Error: 'utf-8' codec can't decode byte 0x9d in position 681: invalid start byte

### PR DESCRIPTION
**Pull request by stishkin**

### CODE REVIEW

**Review by InLineR495 — APPROVED** [review]

**Review by InLineR495 — DISMISSED** [review]

File: restler/engine/transport_layer/messaging.py [hunk 1]
```diff
@@ -208,7 +210,16 @@ def _recvResponse(self, req_timeout_sec):
             if len(buf) == 0:
                 return data
 
-            data += buf.decode(UTF8)
+            try:
+                data += buf.decode(UTF8)
+            except Exception as ex:
+                if self.ignore_decoding_failures:
```
Comment (marina-p) [inline restler/engine/transport_layer/messaging.py:216]: >if self.ignore_decoding_failures: [](http://example.com/codeflow?start=16&length=33)

Could you please move this repetitive code to a helper function?

**Comment by marina-p** [comment]
Please also update restler_user_settings.json, test_restler_settings.py, and the documentation file SettingsFile.md

**Review by marina-p — CHANGES_REQUESTED** [review]
:clock1:

**Comment by marina-p** [comment]
> Please also update restler_user_settings.json, test_restler_settings.py, and the documentation file SettingsFile.md

@stishkin Just double checking, did you see this comment?

**Review by marina-p — APPROVED** [review]
:shipit:

**Pull request merged**